api/Dockerfile

```diff
@@ -22,6 +32,10 @@ FROM alpine:3.13
 ENV TURING_PORT "8080"
 ENV TURING_USER "turing"
 ENV TURING_USER_GROUP "app"
+ENV TURING_OPENAPI_BUNDLE_FILE "/app/openapi.bundle.yaml"
+ENV OPENAPICONFIG_OPENAPIBUNDLEFILENAME "openapi.bundle.yaml"
+# there is no swaggerui but there will be minimum openapi.bundle.yaml there.
+ENV OPENAPICONFIG_SWAGGERUICONFIG_SERVINGDIRECTORY "/app"
```

Comment:
Not very satisfied by this so I'll be thinking of a solution for this.

Reply:
Changed this to a runtime helm configuration instead of baking it into the docker image.